docs/targets/solana.rst

```diff
@@ -374,3 +405,13 @@ The usage of system instructions needs the correct setting of writable and signe
 contracts on chain. Examples are available on Solang's integration tests.
 See `system_instruction_example.sol <https://github.com/hyperledger/solang/blob/main/integration/solana/system_instruction_example.sol>`_
 and `system_instruction.spec.ts <https://github.com/hyperledger/solang/blob/main/integration/solana/system_instruction.spec.ts>`_
+
+Known Issues
+____________
+
+1. `Issue #698 <https://github.com/hyperledger/solang/issues/698>`_: Custom errors are not yet supported.
+2. `Issue #887 <https://github.com/hyperledger/solang/issues/877>`_: Solidity *mapping* may consume all the storage capacity due to a bug that keeps inserting new entries, even if we
+   are only checking if a key exists.
+3. `Issue #932 <https://github.com/hyperledger/solang/issues/932>`_: Multidimensional dynamic arrays are not properly implemented and Solang might fail during compilation.
+4. `Issue #1192 <https://github.com/hyperledger/solang/issues/1192>`_: The data account layout is not yet specified in the Anchor IDL.
```

Comment:
Remove this, people can find the issues on github themselves, and also we don't want to maintain this list - it will get out of date as we resolve issues.

Reply:
I agree people can find this on Github, but they won't read all the open issues we have. I just wanted to highlight things that should be avoided by now.

Reply:
We don't want to put people off.

Reply:
> We don't want to put people off.

This won't put people off, but making them spending time trying out things we know won't work might disappoint them.

Reply:
This was requested in #786

Reply:
Removed

Reply:
I've looked at again at #786 and I think what the ask is a list of things that currently don't work with solang. That is not the same as a list of known issues in github.
What is much more important is a list of things which currently just don't work, for example:
 - `msg.sender` does not work
 - value transfer with function call does not work
 - many yul builtins/opcodes don't work
 - external calls need more information (accounts)
 - erc20 
 - solidity try/catch 
 - solidity error definitions/revert with error definition
 
That's all I can think of right now. 

It might be reasonable to include a link to any issues in github with the label `solana`

Reply:
All those points are important information to add.

I still have the impression, thought, that #786 wants things that are not working (issues):
`Known issues / things which are not working currently (i.e. which solidity syntax/features/functions/apis do not currently work)`

Reply:
Updated